src/utilities.cpp

```diff
@@ -59,7 +59,7 @@ void pasteFilesFromClipboard(FmPath* destPath, QWidget* parent) {
   if(data->hasFormat("x-special/gnome-copied-files")) {
     // Gnome, LXDE, and XFCE
     QByteArray gnomeData = data->data("x-special/gnome-copied-files");
-    char* pdata = gnomeData.data();
+    const char* pdata = gnomeData.constData();
```

Comment:
I forgot to tell this: `man strchr` says `char *strchr(const char *s, int c);` and I only saw the problem when I tried to compile.